tests/common/globals.py

```diff
@@ -0,0 +1,39 @@
+"""
+   Module covers Android & iOS screens' global contents
+"""
+
+import yaml
+
+
+class Globals:
+    """
+    Contains all global level contents, accessible in Pages & Tests
+    """
+
+    AUT_PACKAGE_NAME = 'org.openedx.app'
+
+    def __init__(self):
+        # Read user_preferences.yml and set globals accordingly
+        self.setup_global_environment()
+
+        # CAPABILITIES
+        self.ios_device_name = 'iPhone 14'
+        self.android_device_name = 'Android Phone'
+
+    def setup_global_environment(self):
+        """
+        set environment and read user_preferences for local run
+        """
+
+        with open("./tests/user_preferences.yml") as user_file:
+            user_preferences = yaml.safe_load(user_file)
+            user_file.close()
+
+        self.target_environment = user_preferences.get('Settings').get('target_environment')
```

Comment:
If you want to avoid error in case Settings is not defined use this line instead.
self.target_environment = user_preferences.get('Settings', {}).get('target_environment')
Comment

Reply:
fixed